pkg/couchdb/couchdb.ts

```diff
@@ -0,0 +1,48 @@
+// connect to database
+const res = await fetch("http://127.0.0.1:5984/");
+const textData = await res.text();
+console.log(textData);
+
+// create a database
+
+/** 
+ * @TODO handle all error cases 
+ * 1 - database already exists
+*/
+export const createDatabase = async (dbName: string,
+    couchdbURL: string) => {
+        const database = await fetch(
+            `${couchdbURL}/${dbName}`,
+            { method: "PUT" }
+        );
+
+    const databaseRes = await database.text();
+    console.log(databaseRes);
+}
+
+// insert bulk docs
+/**
+ * @TODO handle all cases
+ *
+ * 201 Created – Document(s) have been created or updated
+ * 400 Bad Request – The request provided invalid JSON data
+ * 404 Not Found – Requested database not found
+ */
+
+/** @TODO type docs array according to input docs columns */
+export const bulkDocs = async (
```

Comment:
Let's include the `HTTP` verb we're using in the function name: `postBulkDocs` better suggests what this funciton does.